### DISCUSSION TITLE
Year Range menggunakan sliderInput tidak berfungsi

### DISCUSSION THREAD

**Discussion by Nava2022** [teamalgoritma/community#102] [Q&A] [answered]
Halo,

Mau tanya untuk tugas Capstone. Dalam salah satu page, saya menggunakan 2 macam widget: selectizeInput untuk pilihan "select group name" dan sliderInput untuk pilihan "select year". Fungsi selectInput utk memilih group name sudah mengahasilkan chart yg diinginkan. Tapi utk sliderInput nya belum berhasil, waktu digeser-geser range tahunnya, chart nya tidak berubah sama sekali. 

Kira2 di mana salahnya? Terimakasih.

Salam,
Nava

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168312322-0309c956-b77b-4b85-8cac-bf17a9146290.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168306868-7689c84f-6627-40be-8ba0-704fef429bca.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168306909-11468178-fa93-44d0-b2c1-c46a0bc2080d.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168306959-07299bd2-439a-487d-8dec-cd769c2aa9ee.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168307342-8f49cc42-7003-4d46-8017-1e61e5eb2854.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168307396-4f7f6498-936c-4bc7-8f74-e131f510e190.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168307527-01fb62b5-cc2f-4460-a65d-c59006a37fc3.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168307572-30094631-82e3-4d36-9c96-33ea01e3d52d.png">

**Comment by Nava2022** [comment, marked as answer]
Saya coba pakai code ini dan sepertinya berhasil. Apakah benar? 
Salam,
Nava

Shiny link: https://nava20.shinyapps.io/Exercise_GlobalTerrorism/

ui.R
sliderInput(inputId = "input_range", min = 1970, max = 2017, 
                          step = 1, sep = '', 
                          value = c(1970, 2017),
                          label = "Select Year Range"
                     
           )

server.R
output$plot5_attype_byg <- renderPlotly({
    attype_byg <- 
      GTD %>%  
        filter (gname == {input$input_gname}& between(iyear, input$input_range[1], input$input_range[2]))
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/168337071-9e708dac-2218-4194-966b-11d11413814b.png">

**Reply by inytss** [reply]
Dear @Nava2022 ,
Cara yang Ibu gunakan sudah tepat yaitu menggunakan between() untuk mengakses nilai minimum dan maksimum dari slider input tersebut.
